doc/classes/PathFollow3D.xml

```diff
@@ -9,6 +9,16 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="correct_posture" qualifiers="static">
+			<return type="Transform3D" />
+			<param index="0" name="transform" type="Transform3D" />
+			<param index="1" name="rotation_mode" type="int" enum="PathFollow3D.RotationMode" />
+			<description>
+				Correct the [code]transform[/code] according to different [code]rotation_mode[/code]
```

Comment:
It would be better to explain that rotation_mode implicitly specifies up_vector and forward_vector and that the posture is calculated.

Also, the sentence should end with a period, but the style check will be done by someone else.

Reply:
done